orchestration/Factory/ImportModules.ps1

```diff
@@ -79,7 +79,7 @@ $scriptBlock = ". $modulePath";
 $script = [scriptblock]::Create($scriptBlock);
 . $script;
 
-$modulePath = Join-Path (Join-Path (Join-Path $rootPath -ChildPath '..') -ChildPath 'OrchestrationService') -ChildPath 'ArchetypeInstanceBuilder.ps1'
+$modulePath = Join-Path (Join-Path (Join-Path $rootPath -ChildPath '..') -ChildPath 'OrchestrationService') -ChildPath 'ConfigurationBuilder.ps1'
```

Comment:
Can you please use -AdditionalPath array instead? In an upcoming PR I am updating all these nested Join-Path with -AdditionalPath.

Reply:
Updated.